docs/index.bs

```diff
@@ -683,6 +702,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
     </pre>
     <pre class="idl" id="registration-option-list-dictionary">
       dictionary RegistrationOptions {
+        DOMString id;
```

Comment:
Please write a test with isolated surrogates to ensure implementations do not use UTF-8 to store these strings. (If you decide to change this to USVString to allow implementations to do that it's probably still worth writing a test.)